opensrp-app/src/main/java/org/smartregister/view/activity/FormConfigurationJsonFormActivity.java

```diff
@@ -70,7 +74,7 @@ public void run() {
                 });
     }
 
-    @NonNull
+    @Nullable
```

Comment:
Hope you are checking for nulls in the places where these methods are used

Reply:
Yea